exonum-java-binding/package_app.sh

```diff
@@ -137,6 +123,23 @@ cp ./core/rust/exonum-java/log4j-fallback.xml "${PACKAGING_ETC_DIR}"
 # Copy tutorial
 cp ./core/rust/exonum-java/TUTORIAL.md "${PACKAGING_ETC_DIR}"
 
+# We use static linkage for RocksDB because in case of dynamic linking
+# the resulting app has a dependency on a _particular_
+# `install_name` of the RocksDB library. In case of RocksDB, `install_name`
```

Comment:
What is the Linux alternative to `install_name` (I'd specify that once and go on using one of the terms)?

Reply:
It is called `SONAME`. I'd perhaps change to `SONAME` because Linux is definitely much more popular.